sources/sqlserver/infoschema.go

```diff
@@ -199,7 +199,7 @@ func (isi InfoSchemaImpl) GetTables() ([]common.SchemaAndName, error) {
 	FROM sys.tables AS TBL
 	INNER JOIN sys.schemas AS SCH 
 	ON SCH.schema_id = TBL.schema_id
-	WHERE TBL.type = 'U' AND TBL.is_tracked_by_cdc = 0 AND TBL.is_ms_shipped = 0 AND TBL.name <> 'sysdiagrams'
+	WHERE TBL.type = 'U' AND TBL.is_ms_shipped = 0 AND TBL.name <> 'sysdiagrams'
```

Comment:
Wondering why this check was added in the first place. Is there any repercussion of removing this check ?

Reply:
Not sure about that, I have tested a POC migration for both CDC and non-CDC enabled tables though to test that the fix works.